smallrye-reactive-messaging-mqtt/src/main/java/io/smallrye/reactive/messaging/mqtt/MqttSink.java

```diff
@@ -38,6 +38,7 @@ public class MqttSink {
     private final SubscriberBuilder<? extends Message<?>, Void> sink;
 
     private final AtomicBoolean started = new AtomicBoolean();
+    private final AtomicBoolean alive = new AtomicBoolean();
```

Comment:
Am I missing something or alive flag is not used later?

Reply:
You right, let me fix it... it was very late yesterday :)